packages/docsify-server-renderer/index.js

```diff
@@ -1,32 +1,32 @@
 import { readFileSync } from 'fs';
 import { resolve, basename } from 'path';
+import resolvePathname from 'resolve-pathname';
+import fetch from 'node-fetch';
+import debug from 'debug';
+import DOMPurify from 'dompurify';
```

Comment:
@anikethsaha How are you making changes? I'm just curious, because this change (re-ordering of imports) was done automatically during `git commit` by the `husky` hook, which should've happened for you too.

In general I agree with the practice of putting 3rd party imports first. It sometimes prevents (perhaps surprisingly) circular dependency issues.

Reply:
particularly I don't have any preference for having an import order. Yes, it should be from husky but for a while, in my vscode there is some conflict with the prettier config. not sure what.